scripts/run-jenkins.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash -xe
+
+TIMEOUTCMD=scripts/babysitter
+export MONO_BABYSITTER_LOG_FILE=babysitter_report.json_lines
+
+if [[ ${label} == debian-arm* ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; fi
+if [[ ${label} == 'osx-i386' ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no --build=i386-apple-darwin11.2.0"; fi
+if [[ ${label} == 'osx-amd64' ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no"; fi
```

Comment:
We do have a couple diverging scripts in the various jobs, e.g. the community arch jobs do something different for windows etc. We should unify these.